src/debugAdapter/goDebug.ts

```diff
@@ -431,12 +431,11 @@ class Delve {
 						if (launchArgs.buildFlags) {
 							runArgs.push(launchArgs.buildFlags);
 						}
+						runArgs.push(program);
 						if (isProgramDirectory) {
 							runOptions.cwd = program;
```

Comment:
in both cases (isProgramDirectory is true or false) `runOptions.cwd` can be `dirname`.

Reply:
you're right, i have moved the `cwd` to initialization of `runOptions` https://github.com/microsoft/vscode-go/pull/3147/commits/763acb860eb30a36e786250a6805e474e8756949